docs/src/docs/asciidoc/user/index.adoc

```diff
@@ -23,6 +23,7 @@ Each topic below corresponds to a menu item at the left.
 |link:xml.html[XML Configuration]|How to configure Ehcache using XML
 |link:usermanaged.html[User Managed Caches]|Creating and making use of caches directly
 |link:107.html[JSR-107 Support]|Using Ehcache as a javax.cache aka JSR-107 provider
+|link:tiering.html[Tiering options]|Using other resources than heap memory, combining them or not
```

Comment:
Not a native English speaker but that doesn't sound right to my Indian ears. How about "Using resources other than heap memory"?